### PR TITLE
[SRS] Add FRs for employee management

### PR DESCRIPTION
**Pull request by LeonSo7**
[SRS]
- Add employee management subsection
- Add FRs for employee invitation
- Add FRs for viewing employees
- Add FRs for searching employees
- Add FRs for removing employees

Depends on [PR#95](https://github.com/arkinmodi/project-sayyara/pull/95) for BE numbering.

### CODE REVIEW

File: docs/SRS/SRS.tex [hunk 1]
```diff
@@ -325,6 +326,30 @@ \subsubsection{Authentication}
 	      \end{enumerate}
 \end{enumerate}
 
+\subsubsection{Shop Management}
+\begin{enumerate}
+	\item [{BE}1.] The shop owner wants to invite and employee to their shop
```
Comment (joyxiao99) [inline docs/SRS/SRS.tex:331]: 'invite an employee'

**Comment by LeonSo7** [comment]
Do not merge until #95 is merged.

File: docs/SRS/SRS.tex (hunk 1 above)
Comment (LeonSo7) [reply docs/SRS/SRS.tex]: Fixed, thanks for catching that.

**Review by arkinmodi — CHANGES_REQUESTED** [review]

File: docs/SRS/SRS.tex [hunk 2]
```diff
@@ -325,6 +326,89 @@ \subsubsection{Authentication}
...
+			            \item The system shall allow the user to enter employee email(s) to invite
+			            \item The system shall send an invitation email to the invited employee(s)
+		            \end{enumerate}
+	      \end{enumerate}
+	      \begin{enumerate}
+		      \item Viewpoint: Auto Repair Shop Employee
+		            \begin{enumerate}
+			            \item The system shall allow the user to accept an invitation
+		            \end{enumerate}
+	      \end{enumerate}
```
[the remark this answers is not in the document]
Comment (LeonSo7) [reply docs/SRS/SRS.tex]: Updated

**Review by arkinmodi — APPROVED** [review]

**Pull request merged**